utils/rpms/daos.spec

```diff
@@ -357,6 +357,8 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %{_bindir}/daos_agent
 %{_bindir}/dfuse
 %{_bindir}/daos
+%{_bindir}/daos_new
+%{_libdir}/libdaos_cmd_hdlrs*
```

Comment:
What is the reason of this wildcard/* sign here ?

Reply:
This is because the libraries are versioned which means that there are multiple symlinks pointing to the main .so (e.g. libdaos_cmd_hdlrs.so.1, libdaos_cmd_hdlrs.so.1.3.0, etc).

Reply:
Hmm. This is wrong. Looking at the specfile again this is different from the other lib handling. Must have been something I pulled in from the debian packaging.